FirstProject/assets/evaluation.less

```diff
@@ -0,0 +1,54 @@
+//Evaluation less file
+
+.row {
+
+  .navigation_bar {
+    border-bottom: 2px solid darkgrey;
+    padding-top: 12px;
+    padding-bottom: 14px;
+  }
+
+  .logo_image {
+    width: 40%;
+  }
+}
+
+.col-md-4 {
+
+  .footer_description {
+    padding: 25px 0 12px 0;
```

Comment:
I was referring to another way of writing it, but such that the values still match what you have. More to the point, both `margin` and `padding` allow 4 different shorthand forms:
* `padding: x;` - all paddings (top, right, bottom, left - remember the specific order) will have a value of `x`
* `padding: x y;` - top and bottom paddings will have a value of `x`, while left and right will have a value of `y`
* `padding: x y z;` - top padding will have a value of `x`, left and right will have a value of `y`, while bottom will have a value of `z`
* `padding: x y z t;` - top padding will be `x`, right padding will be `y`, bottom padding will be `z`, left padding will be `t`

So, in your case, `padding: 25px 0 12px;` will be identical AND it will save you from having to type 2 extra characters! :wink: